webpack.config.js

```diff
@@ -58,6 +59,7 @@ module.exports = [
     module: {
       loaders
     },
+    devtool: 'source-map',
```

Comment:
For performance reasons, I would prefer `cheap-module-source-map` --> https://webpack.js.org/configuration/devtool/#devtool